### PR TITLE
38-currency-decimal-display

### PR DESCRIPTION
**Pull request by Dan-Quin**
Fixes the decimal display errors.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const App = () => {
           </Box>
           <Box smallIcon={"B"}>
             <p>Number of Bits: {gameData.numBits}</p>
-            <p>Curency Amount: {gameData.currencyAmount}</p>
+            <p>Curency Amount: {Number(gameData.currencyAmount.toFixed(1))}</p>
```
Comment (josephmaxwellhyland) [inline frontend/src/App.tsx:32]: This change looks good!

**Review by josephmaxwellhyland — COMMENTED** [review]
The actual fix looks good. However, there are currently conflicts in the **App.tsx** file - the "Currency" spelling has been reverted back to "Curency" instead.

You will want to [merge the **main** branch into this branch](https://hylandtechoutreach.github.io/app-internship-2024/ContributionGuide.html#pulling--merging-from-the-main-branch) to get the latest changes in there.

Make sure to grab the [latest version of VS Code](https://code.visualstudio.com/download) to make the merging process a little easier.

**Review by jvgusch — APPROVED** [review]
Perfect! Once you get your main merged in this change is good to go!

**Review by josephmaxwellhyland — APPROVED** [review]
Looks good!

**Pull request merged**